gcp_variant_transforms/transforms/merge_variants.py

```diff
@@ -42,7 +42,8 @@ def _map_by_variant_keys(self, variant):
     for key in self._variant_merger.get_merge_keys(variant):
       yield (key, variant)
 
-  def _merge_variants_by_key(self, (key, variants)):
+  def _merge_variants_by_key(self, xxx_todo_changeme):
+    (key, variants) = xxx_todo_changeme
```

Comment:
`xxx_todo_changeme` needs to be fixed.

Reply:
I modified it in future PR, but I'll do it here.